src/mappings/util.ts

```diff
@@ -65,8 +75,24 @@ export const findTopLevelCommas = (
   }
   if (commas.length === 0) {
     throw new Error(
-      `No top level comma found in ${text}, it probably isn't a map`
+      `No top level comma found in ${text}, it probably isn't a map`,
     );
   }
   return commas;
 };
+
+export const formatParams = (
+  params: HarvesterLikeCallArgs,
+  formatKey = true,
+): Record<string, any> => {
+  const obj: Record<string, any> = {};
+  params.forEach((p) => {
+    obj[formatKey ? snakeToCamelCase(p.name) : p.name] =
```

Comment:
was this just copied from the other branch?

Reply:
Most of them yes but this format util was brought over from ST Viewer repo, we used it to format GQL results to key/value object. Here I'm using it to do the same to the params and send it to map function.

Reply:
I copied stuff from subquery branch to make merging later easier. We can ignore most stuff here and mainly just take the new functions and plug them in worst case.

Reply:
Would also need `CallIdEnum` from `mappings/entities/common.ts`.